test/test_images/performance/dev.config

```diff
@@ -4,9 +4,9 @@
 # mako update_benchmark test/performance/broker-latency/dev.config
 
 project_name: "Knative"
-benchmark_name: "Development - Broker Latency & Throughput"
-description: "Measure latency and throughput of the broker using various channels."
-benchmark_key: '5903682180743168'
+# benchmark_name: "Development - Broker Latency & Throughput"
```

Comment:
So we are not going to put this config in `kodata`?
Also, should these lines be uncommented?

Reply:
We don't have a kodata-based solution (yet) that allows the same aggregator image to be used for multiple benchmarks. This change unblocks eventing to use multiple benchmarks in the meantime.

Reply:
@grac3gao I think we should move the dev.config out of this directory and into test/performance/broker-imc. Then these lines should be uncommented.

Reply:
done